src/Auth0Client.ts

```diff
@@ -695,6 +696,29 @@ export default class Auth0Client {
     return !!user;
   }
 
+  /**
+   * ```js
+   * await auth0.buildLogoutUrl(options);
+   * ```
+   *
+   * Builds a `/v2/logout` URL for logout using the parameters provided as arguments.
+   * No session data is removed.
```

Comment:
I am not sure we should explicitly mention it is not removing session data. I prefer to mention what a method is doing, not what it is not doing

What do you think @stevehobbsdev ?

Reply:
Agreed. It's just building a URL, we don't really need to say much more than that. I think the method name already sets that expectation.

Reply:
Fixed in a7197e0